src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -76,47 +75,55 @@ public Class<? extends Number> getReturnType() {
 	@Override
 	@Nullable
 	public Class<?>[] acceptChange(final ChangeMode mode) {
```

Comment:
This should check for chunks (getExpr().getReturnType()) and print error soemthing like "Can't set X/Z coords of chunks"

Reply:
I tried this but `getExpr().getReturnType()` is Object.class for some reason, yet I'm adding it